includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -285,6 +285,17 @@ public function setup_wizard_footer() {
 	 */
 	public function setup_wizard_steps() {
 		$output_steps = $this->steps;
+
+		// Hide the activate step if Jetpack is already active, unless WooCommerce Services
+		// features are selected, or unless the Activate step was already taken.
+		if (
+			class_exists( 'Jetpack' ) && Jetpack::is_active() &&
+			empty( array_filter( $this->wc_setup_activate_get_feature_list() ) ) &&
+			'yes' !== get_transient( 'wc_setup_activated' )
```

Comment:
Is this the right comparison? It looks like it will hide the activate step if it has not yet been taken.

Reply:
It will (if Jetpack is already connected), until/unless something requiring WCS has been chosen. Desirable, or not really?